### ISSUE TITLE
Article scrolling via touch events leads to unexpected exception

### ISSUE THREAD

**Issue by quarthex** [grommet/grommet#1147]
### Expected Behavior

Scrolling fast on an Article component shouldn't raise an exception.

### Actual Behavior

💣💥

### URL, screen shot, or Codepen exhibiting the issue

```es6
<Article scrollStep={false} controls={false}>…</Article>
```

### Steps to Reproduce

 1. Using the device mode of Chrome Developer Tools (for instance).
 2. Scroll fast enough to produce a next/previous jump.

### Your Environment

* Grommet version: 1.2.0
* Browser Name and version: Google Chrome 55.0.2883.87
* Operating System and version (desktop or mobile): desktop Linux 64 bits

**Comment by quarthex** [comment]
I added a test case that fails: https://travis-ci.org/quarthex/grommet/builds/194268140#L484

**Comment by alansouzati** [comment]
we need a safe check here for testing purposes `if (childElement)`.

There is no actual DOM in the Jest setup we did. This is why you are getting this "null pointer"

**Comment by quarthex** [comment]
In the test I provided, I used enzyme to mount the component in a fake DOM.
I used the full DOM rendering API in order to propagate the event through the node tree.
It is the tool recommended by React team : https://facebook.github.io/react/docs/test-utils.html.

I agree with your solution. I think the onTouchStart and onTouchMove props could also be completely discarded (set to null) from the resulting component in some cases.
